frontend/src/components/admin/assessment-creation/BasicInformation.tsx

```diff
@@ -85,30 +63,16 @@ const BasicInformation = ({
         </FormControl>
 
         <Box width="50%">
-          <Controller
-            control={control}
-            name="grade"
-            render={({ field: { name }, fieldState: { error } }) => (
-              <FormControl isInvalid={Boolean(error)} isRequired>
-                <FormLabel color="grey.400">Grade Level</FormLabel>
-                <Select
-                  name={name}
-                  onChange={handleGradeChange}
-                  options={gradeOptions}
-                  placeholder="Select a grade"
-                  selectedOptionStyle="check"
-                  useBasicStyles
-                  value={
-                    gradeOptions.find(
-                      (option) => option.value === watch("grade"),
-                    ) || undefined
-                  }
-                />
-                <FormErrorMessage>{error?.message}</FormErrorMessage>
-              </FormControl>
-            )}
-            rules={{ required: "Please select a grade" }}
-          />
+          <FormControl isInvalid={!!errors.grade} isRequired>
+            <FormLabel color="grey.400">Grade Level</FormLabel>
+            <ControlledSelect
+              isRequired
+              name="grade"
+              options={gradeOptions}
+              placeholder="Select a grade"
+            />
+            <FormErrorMessage>{errors.grade?.message}</FormErrorMessage>
```

Comment:
Do we wanna pass a custom error message or add it here?
```suggestion
            <FormErrorMessage>{errors.grade?.message && "Please select a grade"}</FormErrorMessage>
```